src/router/index.js

```diff
@@ -112,4 +111,25 @@ router.beforeEach((to, from, next) => {
 Router auth logic end
 */
 
+// set organization in vuex state if the route org parameter is in vuex user organizations array
+router.beforeEach((to, from, next) => {
+  if (
+    typeof to.params.org !== "undefined" &&
+    store.getters["auth/isAuthenticated"]
+  ) {
+    let orgShortcodes = store.state.auth.user.organizations.map(
+      (organization) => organization.shortcode
+    );
+    store.dispatch("auth/setActiveWorkspace", to.params.org);
+    if (!orgShortcodes.includes(to.params.org)) {
+      // redirect to default page without organization id
+      store.dispatch("auth/setActiveWorkspace", to.params.org);
```

Comment:
shouldn't this be setting the workspace to Personal if want to redirect to default page "without organization id"?

Reply:
yeah it should be unsetActiveWorkspace so that it default to empty string - which is the personal workspace